Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/TrustTemplate/TrustTemplateGuidance.cshtml

```diff
@@ -1,143 +1,19 @@
-@page "/task-list/{id:int}/trust-template-guidance"
+@page "/task-list/{id:int}/template-guidance"
 @model Dfe.PrepareConversions.Pages.BaseAcademyConversionProjectPageModel
 @{
    ViewData["Title"] = "Prepare your trust template";
 }
 
 @section BeforeMain
 {
-   <govuk-back-link link-item="@Links.TaskList.Index"/>
+   <govuk-back-link link-item="@Links.TaskList.Index" />
+}
+@if (@Model.Project.IsSponsored)
```

Comment:
I'd rather just have separate pages, but the ticket specified wanting one url slug for both entry points i.e., Voluntary or Sponsored